src/DataExporter.ts

```diff
@@ -93,90 +98,103 @@ export class DataExporter {
     return outHeader;
   }
 
-  private async calculateMassProps(ids: Id64Array): Promise<MassProps> {
+  private async calculateMassProps(ids: Id64Array, geometryCalculationSkipList: string[] = []): Promise<MassProps> {
```

Comment:
The geometryCalculationSkipList was something hacky that we put in place while we were working through some bugs ... we should remove it from this PR as I don't think it's necessary in the main source.